services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        Ticket.objects.bulk_create([
+            Ticket(
+                row=ticket.get("row"),
+                seat=ticket.get("seat"),
+                order_id=order.id,
+                movie_session=MovieSession.objects.get(
```

Comment:
since you have `movie_session_id` in tickets, it's better to use it and don't make unnecessary queries